### DISCUSSION TITLE
Camera Hub logs storing 800gb of data

### DISCUSSION THREAD

**Discussion by CtrlAltAwesome** [elgatosf/elgato#32] [Help]
My camera hub 6 file says it has 782gb worth of data!! Why is it storing so much? is this a bug?
![camera hub](https://github.com/elgatosf/elgato/assets/137355585/b8de28ac-58f2-4a46-846a-6bba3260db0a)

**Comment by ChekTek** [comment]
Hi! 

For better Camera Hub support, I would recommend reaching out to Elgato Support:  http://e.lga.to/help

There are likely repeated logs in the file, if you could share a segment of the log file here, I can pass the issue along to the team.
